resources/docs/simulation/concepts/physics.md

```diff
@@ -30,23 +30,23 @@ _@hash/physics/spring.rs_
 
 This behavior simulates a spring connected between the agent and one of its neighbors. It adds a force to the agent's `force` field proportional to it's distance from a certain neighbor, based on the base length of the spring, the spring constant, and the amount of damping. To run this behavior, the agent must have a `springs` field, which contains an array of objects defined like so:
 
-```javascript
+```json
 {
-  ...
+  "...": "...",
   "behaviors": ["@hash/physics/spring.rs", "@hash/physics/forces.rs"],
   "springs": [
     {
       "agent_id": string, // The agent_id of the connected agent
       "length": number, // The neutral length of the spring
       "k": number, // The spring constant
       "damping": number, // OPTIONAL The damping constant B
```

Comment:
Is this valid or should I change to

```suggestion
      "agent_id": "string", // The agent_id of the connected agent
      "length": "number", // The neutral length of the spring
      "k": "number", // The spring constant
      "damping": "number", // OPTIONAL The damping constant B
```

?

Reply:
@nshlapo